src/adapters/mc/OsConfigResource.c

```diff
@@ -886,15 +889,7 @@ void MI_CALL OsConfigResource_Invoke_GetTargetResource(
 
     if (MI_TRUE == isCompliant)
```

Comment:
>if (MI_TRUE == isCompliant) [](http://example.com/codeflow?start=4&length=27)

We need to discuss about this. Is there a way to avoid changing the NRP here and do it all from inside of the Compliance static lib?